bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpRendererHandler.java

```diff
@@ -1653,7 +1653,7 @@ private void updateMetaDataState(UpnpEntry media) {
             } else {
                 State albumArt = null;
                 try {
-                    albumArt = HttpUtil.downloadImage(media.getAlbumArtUri());
+                    albumArt = HttpUtil.downloadImage(media.getAlbumArtUri().trim());
```

Comment:
Should the condition 5 lines aboe also be changed? What if `media.getAlbumArtUri()` just has whitespace?
``` 
if (media.getAlbumArtUri().isBlank() || .......
```

Reply:
if its just whitespace, then it quietly fails when downloading and sets the channel to undef, which is unchanged from its current behavior, and would be the desired state (if a device actually ever sends that).  `isBlank()` is probably a better choice in that case to have this fail earlier and save some cpu cycles ?  I'll can make that change as well.

Reply:
Although now that i look at it, there's a strange double if statement here thats quite confusing to read.  I'll take a closer look tomorrow to make sure i don't mess something up, the counter argument is this PR at least only changes the failure case when downloading , and does not potentially introduce new breaking changes.

Reply:
I think you could also change to isBlank 5 lines above, but not in the wrapping if. That should stay ifEmpty. But even without that change LGTM.